tensorflow/python/autograph/converters/break_statements.py

```diff
@@ -101,6 +100,17 @@ def visit_While(self, node):
 
       new_while_node = node[1]
       anno.copyanno(original_node, new_while_node, anno.Basic.DIRECTIVES)
```

Comment:
This annotation will need to be copied on the else branch as well (same for the for loop case).

Reply:
Added.